src/components/screens/ScoreSetEditor.vue

```diff
@@ -891,6 +883,8 @@ export default {
     suggestionsForAutocomplete: function (suggestions) {
       // The PrimeVue AutoComplete doesn't seem to like it if we set the suggestion list to [].
       // This causes the drop-down to stop appearing when we later populate the list.
+
+      // list [] doesn't affect dropdown menu. API side search function should return something so that dropdown menu can get data.
```

Comment:
is this a TODO? I can't tell

Reply:
It seems long ago notes. I delete this one.